server/views/admin/users/index.njk

```diff
@@ -0,0 +1,41 @@
+{% extends "../../partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% set pageTitle = applicationName + " - " + pageHeading  %}
+{% set mainClasses = "app-container govuk-body assessments--index" %}
+
+{% block beforeContent %}
+    {{ govukBackLink({
+		text: "Back",
+		href: '/'
+	}) }}
+{% endblock %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
+
+            {{ govukTable({
+                attributes: {
+                    'data-module': 'moj-sortable-table'
```

Comment:
I think if we move to pagination, sorting client side may be moot, but let's leave it as is now